cli/command/command.go

```diff
@@ -228,7 +230,16 @@ func CommonFlags() []cli.Flag {
 
 // Populate updates the specified project context based on command line arguments and subcommands.
 func Populate(context *project.Context, c *cli.Context) {
-	context.ComposeFile = c.GlobalString("file")
+	if len(c.GlobalStringSlice("file")) == 0 {
```

Comment:
I would have written like this (because I don't like `else`'s :stuck_out_tongue_closed_eyes:), but not sure if it's better or not...

```go
	context.ComposeFiles = c.GlobalStringSlice("file")

	if len(context.ComposeFiles) == 0 {
		context.ComposeFiles = []string{"docker-compose.yml"}
		if _, err := os.Stat("docker-compose.override.yml"); err == nil {
			context.ComposeFiles = append(context.ComposeFiles, "docker-compose.override.yml")
		}
	}
```